crates/apollo-compiler/README.md

```diff
@@ -87,14 +88,15 @@ let schema = Schema::parse(input, "document.graphql");
 
 /// In case of validation errors, the panic message will be nicely formatted
 /// to point at relevant parts of the source file(s)
-schema.validate().unwrap();
+schema.validate(ValidationOptions::default()).unwrap();
```

Comment:
This makes the public API feel quite heavy. Do we expect real use cases for changing the recursion limit from its default? As a precedent `serde_json::Deserializer`, doesn’t offer any way to change the amount of its own recursion limit.

Reply:
Yeah I see. I think having a diagnostic limit is potentially more useful than this (in a server it would be totally reasonable to abort after a handful of validation errors). But maybe there's a better way to provide that option. made it hardcoded now